src/js/components/Meter/Circle.js

```diff
@@ -13,12 +13,17 @@ const Circle = forwardRef((props, ref) => {
   const theme = useContext(ThemeContext);
   const width =
     size === 'full' ? 288 : parseMetricToNum(theme.global.size[size] || size);
-  const height =
-    type === 'pie'
-      ? width / 2
-      : parseMetricToNum(theme.global.edgeSize[thickness] || thickness);
+  let height;
```

Comment:
Can we rename this variable to "strokeWidth", changing the passed in prop to "strokeWidth"?